core/templates/dev/head/components/rule_editor_directive.html

```diff
@@ -28,8 +28,10 @@
     </div>
 
     <div ng-if="rule.type == 'FuzzyMatches'">
-      <classifier-panel rule-inputs="rule.inputs" on-training-data-deletion="onDeleteTrainingDataEntry(index)">
```

Comment:
Sorry for late response. I checked whether original approach and it was created for same thing (modify training data) but in different way. In original approach when user clicks on "is similar to" rule, all answers are displayed in button like blocks. When you click on particular answer it gives you choice to assign that answer to some different answer group. So basically it allows you to transfer training data from from one answer group to other.

I think you should take a look at it for better understanding. My approach is different and it does not allow user to transfer inputs in between answer groups.